executor/src/test/java/org/k8loud/executor/service/MapperServiceImplTest.java

```diff
@@ -0,0 +1,81 @@
+package org.k8loud.executor.service;
+
+import data.ExecutionRQ;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.k8loud.executor.action.Action;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class MapperServiceImplTest {
+    private MapperServiceImpl mapperService;
+    private ExecutionRQ executionRQ;
+
+    @BeforeEach
+    public void beforeEach() {
+        mapperService = new MapperServiceImpl();
+        // valid executionRQ
+        executionRQ = ExecutionRQ.builder()
```

Comment:
Potential change:
1. Create a private method with creating such RQ. 
2. Create 2 Parametrized tests with `@MethodSource` adnotation. (valid and invalid ones)
3. Create two private static methods (probably should return `Stream<Arguments>`), one for invalid RQs, parameters and second for correct ones
 

Advantages: 
- each time we create new Action, we can easily add few lines to make sure it is tested

Reply:
Do we need to test each new Action here?
All of them should be mapped in the same way (at least for now afaik).
It may be worth considering when we'll implement some rules.

Reply:
I feel like we need to check if action name is defined or not